app/frontend/src/submissions/components/SubmissionForm/CoordsMap.vue

```diff
@@ -37,6 +38,11 @@ export default {
       map: null
     }
   },
+  beforeCreate () {
+    // If lat/long aren't numbers, try to make them numbers.
+    this.$options.propsData['latitude'] = Number(this.$options.propsData['latitude'])
+    this.$options.propsData['longitude'] = Number(this.$options.propsData['longitude'])
+  },
```

Comment:
could this be in the parent component instead? 
```<coords-map :latitude="Number(latitude)">```

Reply:
That's only a million times better!

Reply:
Yes -  that's 10^6 time better.